documentation/quickstart-testbot.md

```diff
@@ -0,0 +1,97 @@
+# Getting Started with Testbot Worker 
```

Comment:
Just wondering if there is some way to generate the two quick start guides from a single source file. I mean, I love that now we don't have IFs in the guides the end users see, but I wouldn't mind see IFs on the guides "source code", to avoid maintaining two files that have so much in common.

Reply:
There definitely is. We do it in docs all the time. It's a way to build several version of the same documentation using a partial and dictionary. Check this out, https://github.com/balena-io/docs/#using-partials

I thought about this when writing these docs but can't bring the entire docs engine to render mere 2 getting started guides. Maybe there's a simpler solution we can find out there.